faq/managed-inference.mdx

```diff
@@ -0,0 +1,75 @@
+  description: Get answers to the most frequently asked questions about Scaleway Managed Inference.
+content:
+  h1: Managed Inference
+dates:
+  validation: 2025-02-12
+category: ai-data
+productIcon: InferenceProductIcon
+---
+
+## What is Scaleway Managed Inference?
+Scaleway's Managed Inference is a fully managed service that allows you to deploy, run, and scale AI models in a dedicated environment.
+It provides optimized infrastructure, customizable deployment options, and secure access controls to meet the needs of enterprises and developers looking for high-performance inference solutions.
+
+## Where are the inference servers located?
+All models are currently hosted in a secure data center located in Paris, France, operated by [OPCORE](https://www.opcore.com/). This ensures low latency for European users and compliance with European data privacy regulations.
+
+## What is the difference between Managed Inference and Generative APIs?
+- **Managed Inference**: Allows deployment of curated or custom models with chosen quantization and instances, offering predictable throughput and enhanced security features like private network isolation and access control. Managed Inference is billed by hourly usage, whether provisioned capacity is receiving traffic or not.
+- **Generative APIs**: A serverless service providing access to pre-configured AI models via API, billed per token usage.
+
+## Where can I find information regarding the data, privacy and security policies applied to Scaleway's AI services?
+You can find detailed information regarding the policies applied to Scaleway's AI services in our [Data, privacy, and security for Scaleway's AI services](/managed-inference/reference-content/data-privacy-security-scaleway-ai-services/) documentation.
+
+## Is Managed Inference compatible with Open AI APIs?
+Managed Inference aims to achieve seamless compatibility with OpenAI APIs. You can detailed information in the following documentation: [Scaleway Managed Inference as drop-in replacement for the OpenAI APIs](/managed-inference/reference-content/openai-compatibility/).
+
+## What are the SLAs applicable to Managed Inference?
+We are currently working on defining our SLAs for Managed Inference. We will provide more information on this topic soon.
+
+## What are the performance guarantees (vs. Generative APIs)?
+Managed Inference provides dedicated resources, ensuring predictable performance and lower latency compared to Generative APIs, which are a shared, serverless offering optimized for infrequent traffic with moderate peak loads. Managed Inference is ideal for workloads that require consistent response times, high availability, custom hardware configurations or generate extreme peak loads during a narrow period of time.
+Compared to Generative APIs, no usage quota are applied on the number of tokens per second generated, since the output is limited by the GPU Instances size and number of your Managed Inference Deployment.
```

Comment:
```suggestion
Compared to Generative APIs, no usage quotas are applied to the number of tokens generated per second, since the output is limited by the GPU Instance's size, and the number of your Managed Inference deployments.
```